lib/components/MusicScreen/album_item.dart

```diff
@@ -115,7 +115,9 @@ class _AlbumItemState extends ConsumerState<AlbumItem> {
   bool deletableGotUpdated = false;
   bool canDeleteFromServer = false;
 
-  void checkAllowDeleteFromServer() {
+  void checkAllowDeleteFromServer() {    
+    // no need to bother if setting is disabled
+    if (!(box.get("FinampSettings")?.allowDeleteFromServer ?? false)) return;
```

Comment:
I think you forgot the line that gets the `box` variable

Reply:
Its now outside the function on line 112

Reply:
Ah so it was duplicate before. That's why there was no change highlighted ^^